index.js

```diff
@@ -17,6 +17,24 @@ module.exports.pitch = function (request) {
 
 	validateOptions(require('./options.json'), options, 'Style Loader')
 
+	options.hot = typeof options.hmr === 'undefined' ? options.hot : options.hmr;
```

Comment:
Can we use only one option? Example, i think `hmr` is better, because `hot` property is part of `hmr` (and can be misleading).

Reply:
Done